app/javascript/components/molecules/TabsMenu.js

```diff
@@ -46,45 +46,41 @@ const Tab = ({ title, url, active, onKeyDown, onFocus, onClick, linkRefs }) => {
   )
 }
 
-class TabsMenu extends React.Component {
-  tabRefs = arrayUtils.cycle([])
+const TabsMenu = ({ items, currentUrl }) => {
+  const tabRefs = arrayUtils.cycle([])
 
-  handleKeyDown = (e) => {
+  const handleKeyDown = (e) => {
     keyboard.forEvent(e)
-      .on('leftArrow', (e) => this.tabRefs.prev().focus())
-      .on('rightArrow', (e) => this.tabRefs.next().focus())
+      .on('leftArrow', () => tabRefs.prev().focus())
```

Comment:
not super clear on what the value of this is @callaghanc do you have any idea? Seems like if we tab through, then can use the arrow key to switch back and forth between highlighting the tabs? do you know if that was added as an accessibility item? or something else?

Reply:
@akegan yes the idea is to allow the arrow keys to navigate through the tabs, as an accessibility compliance feature. here's the original PL story for creating the tabs: https://www.pivotaltracker.com/story/show/155753715. and here's the story where the tabs were implemented in partners: https://www.pivotaltracker.com/story/show/154785586. both stories link to these accessibility guidelines which explain the arrow key navigation: https://www.w3.org/TR/2017/NOTE-wai-aria-practices-1.1-20171214/examples/menubar/menubar-1/menubar-1.html#.